### PR TITLE
system-tables: add a description for columns

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)
Add descriptions of columns in "COLUMNS","ENGINES" and "KEY_COLUMN_USAGE" tables.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: system-tables/system-table-information-schema.md [hunk 1]
```diff
@@ -189,6 +189,34 @@ CHARACTER_MAXIMUM_LENGTH: NULL
 1 row in set (0.01 sec)
 ```
 
+`columns` 表中列的含义如下：
```
Comment (tangenta) [inline system-tables/system-table-information-schema.md:192]:
```suggestion
`COLUMNS` 表中列的含义如下：
```
Let's make it consistent.

File: system-tables/system-table-information-schema.md [hunk 2]
```diff
@@ -227,6 +255,15 @@ TRANSACTIONS: YES
 1 row in set (0.00 sec)
 ```
 
+`engines` 表中列的含义如下：
```
Comment (tangenta) [inline system-tables/system-table-information-schema.md:258]:
```suggestion
`ENGINES` 表中列的含义如下：
```

File: system-tables/system-table-information-schema.md [hunk 3]
```diff
@@ -267,6 +304,21 @@ POSITION_IN_UNIQUE_CONSTRAINT: NULL
 2 rows in set (0.00 sec)
 ```
 
+`key_column_usage` 表中列的含义如下：
```
Comment (tangenta) [inline system-tables/system-table-information-schema.md:307]:
```suggestion
`KEY_COLUMN_USAGE` 表中列的含义如下：
```

File: system-tables/system-table-information-schema.md [hunk 4]
```diff
@@ -208,7 +236,7 @@ SHOW COLUMNS FROM t1 FROM test;
 
 ## ENGINES 表
 
-`ENGINES` 表提供了关于存储引擎的信息。从和 MySQL 兼容性上考虑，TiDB 会一直将 InnoDB 描述为唯一支持的引擎。
+`ENGINES` 表提供了关于存储引擎的信息。从和 MySQL 兼容性上考虑，TiDB 会一直将 InnoDB 描述为唯一支持的引擎。此外，`engines` 表中其它列值也都是定值。
```
Comment (djshow832) [inline system-tables/system-table-information-schema.md:239]:
```suggestion
`ENGINES` 表提供了关于存储引擎的信息。从和 MySQL 兼容性上考虑，TiDB 会一直将 InnoDB 描述为唯一支持的引擎。此外，`ENGINES` 表中其它列值也都是定值。
```

**Comment by zimulala** [comment]
PTAL @tangenta @djshow832

**Comment by zimulala** [comment]
PTAL @ran-huang

File: system-tables/system-table-information-schema.md [hunk 5]
```diff
@@ -267,6 +304,21 @@ POSITION_IN_UNIQUE_CONSTRAINT: NULL
...
+
+* `CONSTRAINT_CATALOG`：约束所属的目录的名称。该值始终为 `def`。
+* `CONSTRAINT_SCHEMA`：约束所属的数据库的名称。
+* `CONSTRAINT_NAME`：约束名称。
+* `TABLE_CATALOG`：表所属目录的名称。该值始终为 `def`。
+* `TABLE_SCHEMA`：表所属的架构数据库的名称。
+* `TABLE_NAME`：具有约束的表的名称。
+* `COLUMN_NAME`：具有约束的列的名称。
+* `ORDINAL_POSITION`：列在约束中的位置，而不是列在表中的位置。列位置从 1 开始编号。
+* `POSITION_IN_UNIQUE_CONSTRAINT`：唯一和主键约束为空。对于外键约束，此列是被引用的表的键的序号位置。
```
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:317]: `唯一和主键约束为空` 是指 `唯一约束` 和 `主键约束` 吗？

File: system-tables/system-table-information-schema.md [hunk 6]
```diff
@@ -267,6 +304,21 @@ POSITION_IN_UNIQUE_CONSTRAINT: NULL
...
+* `REFERENCED_TABLE_SCHEMA`：约束引用的数据库的名称。目前在 TiDB 中，除了外键约束，其它约束此列的值都为 `nil`。
+* `REFERENCED_TABLE_NAME`：约束引用的表的名称。目前在 TiDB 中，除了外键约束，其它约束此列的值都为 `nil`。
+* `REFERENCED_COLUMN_NAME`：约束引用的列的名称。。目前在 TiDB 中，除了外键约束，其它约束此列的值都为 `nil`。
```
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:320]:
```suggestion
* `REFERENCED_COLUMN_NAME`：约束引用的列的名称。目前在 TiDB 中，除了外键约束，其它约束此列的值都为 `nil`。
```

File: system-tables/system-table-information-schema.md (hunk 5 above)
Comment (zimulala) [reply system-tables/system-table-information-schema.md:317]: 嗯

**Comment by zimulala** [comment]
PTAL @ran-huang

**Pull request merged**